arangodb-net-standard/ApiHeaderProperties.cs

```diff
@@ -12,6 +12,11 @@ public class ApiHeaderProperties
         /// </summary>
         public string TransactionId { get; set; }
 
+        /// <summary>
+        /// Allow read from followers
```

Comment:
Trivial: maybe add a mention of "dirty reads" like you've done elsewhere? e.g. `Allow read from followers ("dirty reads").`

Reply:
Done